database/src/main/java/com/pip/chatbot/model/Forecast.java

```diff
@@ -10,15 +10,37 @@
 @AllArgsConstructor
 @NoArgsConstructor
 public class Forecast {
-    @Getter @Setter public Integer id;
-    @Getter @Setter public LocalDateTime createdOn;
-    @Getter @Setter public LocalDateTime date;
-    @Getter @Setter public float temperatureHigh;
-    @Getter @Setter public float apparentTemperatureHigh;
-    @Getter @Setter public float windSpeed;
-    @Getter @Setter public float pressure;
-    @Getter @Setter public float humidity;
-    @Getter @Setter public String summary;
-    @Getter @Setter public String precipType="Brak opadów";
-    @Getter @Setter public String city;
+    @Getter
+    @Setter
+    public Integer id;
+    @Getter
+    @Setter
+    public LocalDateTime createdOn;
+    @Getter
+    @Setter
+    public LocalDateTime date;
+    @Getter
+    @Setter
+    public float temperatureHigh;
+    @Getter
+    @Setter
+    public float apparentTemperatureHigh;
+    @Getter
+    @Setter
+    public float windSpeed;
+    @Getter
+    @Setter
+    public float pressure;
+    @Getter
+    @Setter
+    public float humidity;
+    @Getter
+    @Setter
+    public String summary;
+    @Getter
+    @Setter
+    public String precipType = "Brak opadów";
+    @Getter
```

Comment:
Usunąć ten default

Reply:
Default jest dlatego, ponieważ API pogodowe, w momencie gdy nie ma opadów nie wysyła nam precipType = 'brak', tylko nie zawiera tego pola w JSONie po prostu, przez co w modelu to pole by było nullem.